ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/SpacingAroundCommaRule.kt

```diff
@@ -19,9 +20,10 @@ class SpacingAroundCommaRule : Rule("comma-spacing") {
         if (node is LeafPsiElement && node.textMatches(",") && !node.isPartOfString()) {
             val prevLeaf = node.prevLeaf()
             if (prevLeaf is PsiWhiteSpace) {
-                emit(prevLeaf.startOffset, "Unexpected spacing before \"${node.text}\"", true)
-                if (autoCorrect) {
-                    prevLeaf.node.treeParent.removeChild(prevLeaf.node)
+                val canBeAutoCorrected = prevLeaf.prevLeaf { it !is PsiWhiteSpace } !is PsiComment
+                emit(prevLeaf.startOffset, "Unexpected spacing before \"${node.text}\"", canBeAutoCorrected)
```

Comment:
Since this is a weird edge case, might be nice to add some more info about why it can't be auto corrected.

Reply:
I added a comment with link to a related issue, is that enough?

Reply:
@z3d1k in-code comments are great but I actually meant the text output by ktlint. Most people will probably be using ktlint as a tool and not looking at the code, and since *most* of the time, this rule can be auto-corrected, it might look like a bug that it doesn't autocorrect in this case. Thoughts?

Reply:
> in-code comments are great but I actually meant the text output by ktlint.
I agree with this sentiment.